src/main/java/seedu/sellsavvy/logic/commands/ordercommands/UnmarkOrderCommand.java

```diff
@@ -31,8 +31,7 @@ public class UnmarkOrderCommand extends Command {
 
     public static final String MESSAGE_UNMARK_ORDER_SUCCESS =
             "The order has been reverted to the pending status: %1$s";
-    public static final String MESSAGE_ORDER_ALREADY_UNMARKED_WARNING = "Note: "
-            + "This order is already in the pending status";
+    public static final String MESSAGE_ORDER_ALREADY_UNMARKED = "The order has already been marked as pending.";
```

Comment:
this feels weird

Reply:
```suggestion
    public static final String MESSAGE_ORDER_ALREADY_UNMARKED = "The order is presently not marked as completed";
```
Consider changing "presently" to something else.